src/components/ont/OntTubeSelectedList.vue

```diff
@@ -63,6 +74,14 @@ export default {
     requestClicked({ id, selected }) {
       this.selectRequest({ id, selected: !selected })
     },
+
+    addTubeToPool(id) {
+      this.selectRequest({ id })
+      this.disabledButtons = {
+        ...this.disabledButtons,
+        [id]: true,
```

Comment:
will look into that.

Reply:
didn't find a proper place to hold the disabled status in store. stick to the local list for now.

Reply:
What I mean is that you can check the ontPools store to determine if a tube is selected, and based on that, you can decide whether a button should be disabled or enabled. In the component, if we check whether the tube is already in selectedTubes, the button can be disabled; otherwise, it can be enabled. I don't think it's necessary to keep the disabled status separately.

Reply:
Ok, I will have another look.

Reply:
Hi Seena,
I have a further look at the store, the selectedTubes is the list in 'Selected Tubes' session, they are ready to be choose to the pool, not the ones which are added to the pool. I couldn't figure out which list is the one which selected to the pool. Could you please point out?

Reply:
@wendyyang Can we have a call?

Reply:
You can use `selectedRequests` in the pool store (in getters). When you add a tube, this array will be populated with the tube request whose `id` is the same as the row `id` in the table. Therefore, in the component, we can check whether an element of that id exists to enable or disable it.
 Something like
this.selectedRequests.find((request) => request.id === row.item.id)

Reply:
> @wendyyang Can we have a call?

Sorry, I went for BBQ just now. Thanks for your tips, will have a go.

Reply:
In case of any further questions, please let me know.

Reply:
Thank you. It is working now. Please take a look.